src/Common/src/System/Net/SecurityProtocol.cs

```diff
@@ -8,7 +8,11 @@ namespace System.Net
 {
     internal static class SecurityProtocol
     {
-        public const SslProtocols DefaultSecurityProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
+        public const SslProtocols DefaultSecurityProtocols =
+#if !netstandard
```

Comment:
What library builds for netstandard and includes this file?

Reply:
System.Net.Http.WinHttpHandler

```xml
<PropertyGroup>
  <PackageConfigurations>
    netstandard-Windows_NT;
    netstandard;
    net461-Windows_NT;
  </PackageConfigurations>
  <BuildConfigurations>
    $(PackageConfigurations)
    netfx-Windows_NT;
  </BuildConfigurations>
</PropertyGroup>
```

But, I agree, it seems weird that they still do.

Reply:
@davidsh, @karelz, do we still need to build this package for netstandard?

Reply:
But, @bartonjs, don't block on that.  We can clean this stuff up after if we're able to.

Reply:
>@davidsh, @karelz, do we still need to build this package for netstandard?

We still need to build System.Net.Http.WinHttpHandler package.  It is used by developers on .NET Framework since it supports HTTP/2.

Now, it currently builds against 'netstandard' and not 'net462' directly.  Not sure why we did that but I think it was a relic when we were building separate packages for everything.

We could consider changing how the WinHttpHandler package gets built.  We could just build it for 'net462' and then developers can still use it for .NET Framework based scenarios.